opentracing-util/src/test/java/io/opentracing/util/GlobalTracerTestUtil.java

```diff
@@ -59,6 +59,11 @@ public static void setGlobalTracerUnconditionally(Tracer tracer) {
             globalTracerField.setAccessible(true);
             globalTracerField.set(null, tracer);
             globalTracerField.setAccessible(false);
+
+            Field isRegisteredField = GlobalTracer.class.getDeclaredField("isRegistered");
```

Comment:
This does something that is not according to the method name.
Setting unconditionally would intuitively leave `isRegistered` to true.
_Resetting_ should clear it though.

Reply:
I agree with @sjoerdtalsma - IHMO resetting should be the only one touching `isRegistered` (an alternative would be to receive a `isRegistered` parameter, but it's probably not something needed at the moment, and thus could be added later if needed).